tpu/create_msmarco_monot5_input.py

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
Maybe rename the file to "create_msmarco_monot5_inference.py as there is another one called "create_msmarco_monot5_train.py"

Reply:
I can do that
just FYI, "create_msmarco_monot5_input.py" is the same name as in pyserini's repo